yaksh/templates/manage.html

```diff
@@ -15,7 +15,7 @@
    <link rel="stylesheet" href="{{ URL_ROOT }}/static/yaksh/css/manage.css" type="text/css" />
    {% block css %}
    {% endblock %}
-
+<script language="JavaScript" type="text/javascript" src="{{ URL_ROOT }}/static/yaksh/js/jquery-1.4.2.min.js"></script>
```

Comment:
base.html is not included in manage.html. So I added jquery in both the templates base.html and manage.html